asterius/rts/rts.float.mjs

```diff
@@ -231,4 +239,178 @@ export class FloatCBits {
       (BigInt(man_high) * (BigInt(1) << BigInt(32)) + BigInt(man_low));
     return [acc, exp];
   }
+
+  // from cbits/primFloat
+  rintFloat(f) {
+    const bits = this.FloatToIEEE(f);
+    let fexp = BigInt(this.floatExponentFromBits(bits));
+    let fman = BigInt(this.floatMantissaFromBits(bits));
+    let fsign = BigInt(this.floatSignFromBits(bits));
+
+    // put back the float together
+    const reconstructFloat = function() {
+      return this.IEEEToFloat(
+        Number((fsign << BigInt(31)) | (fexp << BigInt(23)) | fman)
+      );
+    }.bind(this);
```

Comment:
Using `.bind(this)` is not necessary, use arrow function for `reconstructFloat`/`reconstructDouble`, and their `this` will be fixed to the scope of their definition site.